src/components/Pin.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
-const Pin = ({ imageUrl, alt }) => (
-  <div className="map-icon bounce">
+const Pin = ({ imageUrl, alt, onClick }) => (
+  <div className="map-icon bounce" onClick={onClick} aria-hidden="true">
```

Comment:
TIL `aria-hidden`:  Oh! It's interesting! but I feel we need to take care about using this property because some elements can be removed from the accessibility tree. We need to read more about it.

Reply:
Agreed! It's a really good topic to discuss.

One random thing- as the code is currently written, if you remove `aria-hidden="true"` the ability to click on a pin is disabled. 

If we keep `aria-hidden="true"` then we do not have the ability to tab through every pin that appears on the map, and to hear the descriptions of each pin if using Voice Over (at least with the native Voice Over on my MacBook Air).

But. 

If the code was changed to something like the following:

```
const Pin = ({ imageUrl, alt, onClick }) => (
  <div
    className="map-icon bounce"
    onClick
    onKeyPress={onClick}
    role="button"
    tabIndex="0"
  >
```

we can tab around from pin to pin, select the pin by either mouse click or with the keyboard. 

@alxmcr @laurenmbeatty do you think this would be an improvement? There are a lot of pins to tab through for certain map areas, but the point is to look around at everything nearby that is available!

Reply:
Thanks, @jssckbl for share your thoughts!

Well, I just read a few articles (because I'm still learning😄), and I used your code snippet. I have some comments:

1. With your snippet code, the result is the same: you can use the keyword and press `tab` & `enter`. This is awesome.
2. About `aria-hidden`: I read more and I discovered that this property could have some browser & screen reader [compatibility issues](https://www.tpgi.com/html5-accessibility-chops-hidden-and-aria-hidden/). But, `role` & `tabindex` doesn't have.
3. TIL: There is a property `hidden`.

**My humble opinion**: I think it's much better with accessibility topics if you use `role` (because it's a [clickable element](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#clickable_elements_must_be_focusable_and_should_have_interactive_semantics)) and `tabIndex` (it could be helpful to [focus](https://developer.mozilla.org/es/docs/Web/HTML/Global_attributes/tabindex) purpose).

Happy learning 😄!

Reply:
Also, I forgot in your code snippet there was a little typo because onclick is two times😅👀.

Reply:
I guess, I'm wondering why you decided to use `aria-hidden` in the first place. What was the reason? If `aria-hidden=true` isn't there, then you can't click on it? Is that what you're saying?

Adding `role=button` and a `tabIndex` would add the ability for SR users to tab to and access information, so that's a plus, in my book. There might be a lot of pins to tab through, so you'd want to  add a skip link so that users can skip to the navigation if they'd like, rather than tab through everything...not for this PR, but make a separate ticket.

If something has an `onClick`, maybe it's an actual button and it should just be a button.

My 2cents: add role, and tabIndex (or switch it to a button), then add the skip link later.

Reply:
I don't recall why `aria-hidden` was used. @katia-utochkina do you recall?

I was thinking about the skip link as well. I am also thinking about how much time has been put into this story this week so far :) It gets to be so easy to "just add one more thing", so yes, creating a new PR sounds perfect. I will write that up and hopefully set it up appropriately!

Thank you for all the feedback and conversation on this PR. Time to go get that going on the other pair's PR 😄

Reply:
``aria-hidden`` was used to get rid of red console messages 😄,